src/graph_ops.cc

```diff
@@ -1,55 +1,40 @@
 #include "./graph_ops.h"
 
+#include "cuda/fusion/random_walk.h"
+#include "cuda/fusion/slice_sampling.h"
 #include "cuda/graph_ops.h"
-#include "cuda/heterograph_ops.h"
-#include "cuda/random_walk.h"
+#include "cuda/tensor_ops.h"
 
 namespace gs {
 
-std::shared_ptr<COO> GraphCSC2COO(std::shared_ptr<CSC> csc) {
+std::shared_ptr<COO> GraphCSC2COO(std::shared_ptr<CSC> csc, bool CSC2COO) {
```

Comment:
The `bool CSC2COO` parameter seems not so straightfoward for me. Maybe four operators `CSC2COO`, `CSR2COO`, `COO2CSC`, `COO2CSR` is better.

Reply:
Similar with parameter `reverse` in DGL. If CSC2COO is True, it turn csc to coo. If not, it turn csr to coo